src/track/track.cpp

```diff
@@ -368,6 +368,20 @@ void Track::setDateAdded(const QDateTime& dateAdded) {
     return m_record.setDateAdded(dateAdded);
 }
 
+QDateTime Track::getLastPlayedDate() const {
+    QMutexLocker lock(&m_qMutex);
+    return m_lastPlayed;
+}
+
+void Track::setLastPlayedDate(const QDateTime& playedDate) {
+    QMutexLocker lock(&m_qMutex);
+    if (playedDate != m_lastPlayed) {
```

Comment:
Should this be `==`? If I understand correctly, the purpose is to avoid marking the Track as dirty for a no-op, right?

Reply:
lol yup